### DISCUSSION TITLE
Date Field's empty value constrains day boundry to the current month by default

### DISCUSSION THREAD

**Discussion by writes-of-spring** [adobe/react-spectrum#6602] [Q&A] [answered]
I'm in a dd/mm/yyyy locale, and when entering a date via the date field, I can't enter 31 in an empty field, I'd have to enter a day, then month, then go back to change the day.

Should the day constraint only occur after a month is in place? I do get that it's easier to do with a mm/dd/yyyy format.

have tested and is working like that on the example in https://react-spectrum.adobe.com/react-aria/DateField.html (provided dd/mm/yyyy) is the locale!

Thanks

**Comment by yihuiliao** [comment, marked as answer]
We have an issue open tracking this bug https://github.com/adobe/react-spectrum/issues/3256
